tests/test_aurora.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+import torch
+from graph_weather.models.aurora import (
+    IntegrationLayer,
+    GenCastConfig,
+    Fengwu_GHRConfig,
+    Swin3DEncoder,
+    Decoder3D,
+    LoraLayer,
+    PerceiverProcessor
+)
+
+def test_aurora_integration():
```

Comment:
I like this integration end to end test, although it will need changes once the GenCast and Fengwu-GHR ones are removed. But I would like to also see unit tests for each of the modules you have added, so just the `LoraLayer`, etc. They should be fairly small and simple to run, and should all be able to run on the CPU, so no need for checking the device.